lib/http.hoon

```diff
@@ -0,0 +1,35 @@
+::
```

Comment:
Is this file actually be used by anything, or was it spuriously re-added in https://github.com/urbit/arvo/commit/8eac489504589d23fe3108d9e59b4de4f1097bdb#diff-745c223f983f23c1162e3fc8932996b4 ?

Reply:
spurious re-add

Reply:
[):](https://github.com/Fang-/djay/blob/master/app/djay.hoon#L4)

Are we gonna get an alternative "build http request structure, but slightly easier" lib?

Reply:
Perhaps a `lib/client`, to go with the `lib/server` added in this PR? In any case worth noting that this PR (and the others targeting `lighter-than-eyre`) are just finishing the development of `lighter-than-eyre`. There will be final PRs here and on urbit/urbit, wherein the whole effort should receive (any necessary) critical reviews.